### ISSUE TITLE
About Training Model

### ISSUE THREAD

**Issue by zhangkun07** [flow-diffusion/flow-diffusion#9]
Thank you for your amazing work, I noticed that only the code for the inference phase is provided in this repository and not a copy of the code on how to train the model, could you please provide another copy of the code when the model is trained. Also, I would like to ask if the entire loss function of the model consists of only the two loss functions in subsection 3.1 and 3.3 of the paper. When summing these two loss functions to construct the whole loss function, what are the weight hyperparameters of each loss function?Looking forward to your reply.
